archive/*/objects/MMDSResolve: remove MMDSResolve messages

batrick opened the pull request:
These messages are versioned via CEPH_MDS_PROTOCOL cluster protocol. The
decoder doesn't know how to handle older messages yet.

Signed-off-by: Patrick Donnelly <batrick@batbytes.com>

gregsfortytwo commented:
Yeah, this message embeds the (unversioned) dirfrag_t struct so having these in here seems like a mistake.

batrick committed:
archive/13.0.0/forward_incompat: add MMDSResolve

These messages are versioned via CEPH_MDS_PROTOCOL cluster protocol. The
decoder doesn't know how to handle older messages yet.

Relevant ceph change: ebd0fcd067

Signed-off-by: Patrick Donnelly <batrick@batbytes.com>
